docs/definitions/tenderers_summary.csv

```diff
@@ -1,17 +1,17 @@
 Column Name,Data Type,Description
 id,bigint,"Unique id representing a release, compiled_release or record"
-tenderer_index,bigint,
+tenderer_index,bigint,Position of the tenderer in the tenderer array
 release_type,text,"Either release, compiled_release or record. compiled_release are releases generated by kingfisher release compilation"
 collection_id,integer,id from kingfisher collection table
 ocid,text,ocid from the data
 release_id,text,Release id from the data. Relevent for releases and not for compiled_releases or records
 data_id,integer,"id for the ""data"" table in kingfisher that holds the original JSON data."
-tenderer,jsonb,
-tenderer_parties_id,text,
-tenderer_identifier,text,
-unique_identifier_attempt,text,
-tenderer_additionalidentifiers_ids,jsonb,
-tenderer_additionalidentifiers_count,integer,
-link_to_parties,integer,
-link_with_role,integer,
-party_index,bigint,
+tenderer,jsonb,JSONB of tenderer object
+tenderer_parties_id,text,`id` from tenderer object
+tenderer_identifier,text,Concatenation from `scheme` and `id` from `identifier` object in the form `<scheme>-<id>`
+unique_identifier_attempt,text,"The `id` from tenderer object if exists, otherwise the identifier described above if exists, otherwise the party name"
+tenderer_additionalidentifiers_ids,jsonb,JSONB list of the concatonating scheme and id of all additionalidentifier objects
```

Comment:
the concatonating -> concatenating the?

Reply:
additionalidentifier -> additionalIdentifier (to be consistent with standard docs)?